### DISCUSSION TITLE
Add examples

### DISCUSSION THREAD

**Discussion by vissarion** [GeomScale/volesti#291] [Show and tell]
**Is your feature request related to a problem? Please describe.**
It would be very useful to add examples of all algorithms such as volume computation of H-polytopes, V-polytopes, Z-polytopes, sampling from those and rounding.  Now there are only for spectahedra see https://github.com/GeomScale/volume_approximation/tree/develop/examples

**Describe the solution you'd like**
An example should be ideally simple, clean and descriptive.

**Comment by vaithak** [comment]
Hi @vissarion, If nobody is working on this. I would like to work on this. I will start by adding examples for volume computation of H-polytopes.

**Comment by vissarion** [comment]
This is still an open issue. More examples are needed.

**Comment by antreev-brar** [comment]
I would like to work on this issue

**Comment by vissarion** [comment]
> I would like to work on this issue

@antreev-brar sure, please submit your PR.

**Comment by bsarthak16** [comment]
I would like to work on this issue if it is still open

**Comment by vissarion** [comment]
Sure, you are welcome to propose examples.

**Comment by bsarthak16** [comment]
Add Example For Computing Volume Of Cubes #268

**Comment by eliazonta** [comment]
Hi, I'm new to the project and would like to contribute if possible and if the "issue" is still open 
I'm experimenting with the project and would like to know which examples could be more useful to add before others in order to add something valuable 

tysm and have a good day
